index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Addisalem seifu Portofolio</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+    <header>
+        <nav>
+            <p>
+                My Logo
+            </p>
+            <img src="C:/Users/Wuro/Downloads/img/menue.png" alt="">
```

Comment:
- [ ] Please correct the hamburger button's image link. It is broken.

![image](https://user-images.githubusercontent.com/53019916/125473049-ab7276e4-83ef-458a-8e6e-cf01f64d284e.png)